pkg/db/db.go

```diff
@@ -49,10 +49,10 @@ type Operation interface {
 }
 
 type Metadata struct {
-	Version    int
-	Type       Type
-	NextUpdate time.Time
-	UpdatedAt  time.Time
+	Version    int        `json:",omitempty"`
+	Type       Type       `json:",omitempty"`
+	NextUpdate *time.Time `json:",omitempty"`
+	UpdatedAt  *time.Time `json:",omitempty"`
```

Comment:
I think `NextUpdate` and `UpdatedAt` are always not empty, while `Metadata` itself is sometimes empty. I know you want to omit these values if they are empty, but it doesn't have enough effects here. If possible, I'd like to avoid the pointer because we have to do nil check anywhere. If we forget it, it introduces panic. I mean effort and achievement are not balanced.

Reply:
You are correct, they aren't empty but have an initialized zero value of: `0001-01-01 00:00:00 +0000 UTC`. 

I also agree we should leave them as is and not use pointers. Simple is better. I was trying to make the output look better by removing them if zero. I agree it's not worth the added complexity of maintaining it.